cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -165,7 +165,8 @@ describe('Script Builder', () => {
       cy.wait('@query -15m')
     })
 
-    describe('data completeness', () => {
+    // TODO(wiedld): this test is flaky. Need to debug later.
+    describe.skip('data completeness', () => {
```

Comment:
It was indeterminately passing. Some of the `it()` test blocks passed, and some failed, with each test runs. And they were different it blocks failing each time. A.k.a. flake.

Going to queue this up to debug in a separate PR. Because when we put up PRs with ONLY the tests modified, the CI pipeline re-runs that test suite 20x times. This is a way that we can more effectively debug the flake.